tests/integration/src/xvm.rs

```diff
@@ -0,0 +1,334 @@
+// This file is part of Astar.
+
+// Copyright (C) 2019-2023 Stake Technologies Pte.Ltd.
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+// Astar is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Astar is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Astar. If not, see <http://www.gnu.org/licenses/>.
+
+use crate::setup::*;
+
+use astar_primitives::xvm::{Context, VmId, XvmCall};
+use frame_support::{traits::Currency, weights::Weight};
+use parity_scale_codec::Encode;
+use sp_runtime::MultiAddress;
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `use` statements are importing the required dependencies. No issues found.



<!-- This is an auto-generated comment by OSS CodeRabbit -->